### DISCUSSION TITLE
Is the serverless-workflow specification being actively developed?

### DISCUSSION THREAD

**Discussion by SatheeshJM** [serverlessworkflow/specification#697] [General] [answered]
In the discussion here (https://github.com/serverlessworkflow/specification/discussions/526), the goal seemed to have been to have 1.0 released by July 2022,   buts it's October already and we are still at 0.8.
The last release (v0.8) was almost a. year back.

**Comment by tsurdilo** [comment]
Yes it's being actively developed. 

Our originally expected timelines did move some you are correct. It was important for us to temporarily "slow down" adding features to specification to allow runtimes to implement as  many existing features as possible and provide us their valuable feedback. We are using this feedback to make updates to spec which will help us deliver a better 1.0 release.
We will provide an updated roadmap and currently estimated release timelines for KubeCon NA.

**Reply by cdavernas** [reply]
I'd add that given the need to remodel some key parts of the spec(errors, for one) and given the promise we made that no breaking changes would be introduced after 0.9, we took the decision to delay the latter until all required changes can be pushed on. 
Imho, the delay is a bliss: it will ensure a production/industry-grade 1.0 version, which you'll be able to fully rely on.
For further info, please don't hesitate to join our weekly meetings (check Readme): we address all issues, PRs and upcoming features in a collegial yet informal way.

**Comment by SatheeshJM** [comment]
Thank you . That is good to know!

**Comment by SatheeshJM** [comment]
One question though. You'd mentioned "runtimes".. are there more than 1 runtime? I was only aware of synapse. 
@tsurdilo

**Reply by cdavernas** [reply, marked as answer]
There are multiple implementations out there:
- Automatiko (speak to @mswiderski)
- Kogito (RedHat, speak to @ricardozanini)
- FaasNet (speak to @thabart)
- Synapse (speak to me ;) )

For all I know, said implementations have all a different (valid) approach to SW, and the choice you'll make will probably depend on your requirements.

Note, however, that it should not matter in the end, as SW definitions MUST be 100% portable from runtimes to runtimes, extensions left aside (at least, that's the whole point of the spec).

If you choose on Synapse, know that you are more than welcome to the project, as we are convinced that through open collaboration we will achieve an amazing product (reason why my company gave the project away to SW/CNCF)
